svelte/README.md

```diff
@@ -0,0 +1,48 @@
+# Svelte + TS + Vite
+
+This template should help get you started developing with Svelte and TypeScript in Vite.
+
+## Recommended IDE Setup
+
+[VSCode](https://code.visualstudio.com/) + [Svelte](https://marketplace.visualstudio.com/items?itemName=svelte.svelte-vscode).
+
+## Need an official Svelte framework?
+
+Check out [SvelteKit](https://github.com/sveltejs/kit#readme), which is also powered by Vite. Deploy anywhere with its serverless-first approach and adapt to various platforms, with out of the box support for TypeScript, SCSS, and Less, and easily-added support for mdsvex, GraphQL, PostCSS, Tailwind CSS, and more.
+
+## Technical considerations
+
+**Why use this over SvelteKit?**
+
+- It brings its own routing solution which might not be preferable for some users.
```

Comment:
I think Vite? I just grab the default content/files that are created with minimal changes to enable Clarity 🤷

Reply:
Should we specify anywhere what you used to generate the app? For example, the react folder is called `create-react-app` because that's the generator that was used.

It looks like you used Vite to generate this, but it's not immediately clear? https://vitejs.dev/guide/#scaffolding-your-first-vite-project